.gitignore

```diff
@@ -93,3 +93,6 @@ bundler.d/*
 # yardoc
 /doc/
 .yardoc
+
+# byebug
+.byebug_history
```

Comment:
This feels like a personal ignore as opposed to a project ignore.  As such, our general policy on personal ignores (for things like IDEs, OS files, etc) is that you can use git's global gitignore feature.

See https://git-scm.com/docs/gitignore:
> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

Reply:
Thanks for the info, I have now removed those lines from the gitignore